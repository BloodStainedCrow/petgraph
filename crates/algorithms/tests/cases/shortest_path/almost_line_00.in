version https://git-lfs.github.com/spec/v1
oid sha256:ddcc091c3d39eef908e4de002e1b498778ad8230111c524896f4b0f7c6e9230d
size 11500830
