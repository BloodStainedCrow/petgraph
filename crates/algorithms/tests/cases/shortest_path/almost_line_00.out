version https://git-lfs.github.com/spec/v1
oid sha256:46a253236925af5a56ca1af935ff2abc65b3a4dbe1495f55d5eb8287d590a584
size 2033784
